tools/sgbiome/tools.go

```diff
@@ -0,0 +1,63 @@
+package sgbiome
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"runtime"
+
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/sgtool"
+)
+
+const (
+	toolName = "biome"
+	version  = "v1.6.0"
+)
+
+func Format(ctx context.Context, paths ...string) error {
+	sg.Deps(ctx, prepareCommand)
+
+	args := []string{
+		"format",
+		"--write",
+		"--log-kind",
+		"compact",
+	}
+	binDir := sg.FromToolsDir(toolName, version)
+	execDir := filepath.Join(binDir, "biome-linux-x64")
```

Comment:
what if this is downloaded on a mac, then it wont be linux, should follow the format they have for different platforms
![image](https://github.com/einride/sage/assets/102452/3afcd8e4-09b7-4f05-82dd-41a04a054ae5)

Reply:
ah, I thought I had fixed that. fixing fixing

Reply:
I think you can copy-paste most from this setup, https://github.com/einride/sage/blob/master/tools/sggolines/tools.go